atmo/decorators.py

```diff
@@ -67,3 +67,29 @@ def change_permission_required(model, **params):
 
 def delete_permission_required(model, **params):
     return permission_required(full_perm(model, 'delete'), model, **params)
+
+
+def modified_date(view_func):
```

Comment:
Yeah, Etag headers are similar, except that I can't do complex comparison on them and Last-Modified headers have a weird format that I can't reliably parse on the JS side with Moment. So this invents its own header :(